File: ICS_101/practice/final/pyramid/displaystarpyramid.dart
import 'dart:io';

void main() {
  int n;

  stdout.write('Enter the number of rows: ');
  n = int.parse(stdin.readLineSync() ?? '0');

  for (int i = 0; i < n; i++) {
    print('\n');
    for (int j = 0; j < n - 1; j++) stdout.write(' ');
    for (int j = 0; j < 2 * i - 1; j++) stdout.write('*');
  }
}

File: ICS_101/practice/final/pyramid/displaystarpyramid.go
package main

import "fmt"

func main() {
	var i, j, n int

	print("Enter the number of rows: ")
	fmt.Scan(&n)

	for i = 1; i <= n; i++ {
		for j = 1; j <= n-i; j++ {
			fmt.Print(" ")
		}
		for j = 1; j <= 2*i-1; j++ {
			fmt.Print("*")
		}
		fmt.Println()
	}

}
